utils/getQueryObject.tsx

```diff
@@ -0,0 +1,27 @@
+import { queryObject } from "interfaces/queryObject";
+
+/**
+ * Combines new and existing query params, removing empty key-value pairs when no value is provided for a given query params. If only one argument is provided, removes empty key-value pairs.
+ * @param currentParams : existing query parameters
+ * @param newParams : new query parameters
+ * @returns : updated query parameters with no empty key-value pairs
+ */
+
+export function getQueryObject(
```

Comment:
Let's have a chat about this.

Reply:
Not clear what the function does, consider remove.